source/secure_tunneling.c

```diff
@@ -35,7 +34,6 @@ static void s_on_websocket_setup(
     aws_http_message_release(secure_tunnel->handshake_request);
     secure_tunnel->handshake_request = NULL;
 
-    secure_tunnel->stream_id++;
```

Comment:
This function is called when the websocket is established. We shouldn't change `stream_id` here.